sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/http/BufferedHttpResponse.java

```diff
@@ -53,6 +55,11 @@ public HttpHeaders headers() {
         return bodyAsByteArray().flatMapMany(bytes -> Flux.just(Unpooled.wrappedBuffer(bytes)));
     }
 
+    @Override
+    public Flux<InputStream> bodyAsInputStream() {
+        return bodyAsByteArray().flatMapMany(bytes -> Flux.just(new ByteArrayInputStream(bytes)));
```

Comment:
why do we need `bodyAsInputStream()` ? why can't we fully rely on the materialized String or Bytes?

whey you wrap in ByteArrayInputStream memory will be GCed later. this kind of is similar to inmobi issue

Reply:
Good point, and since attachment is not used at all, I will try to get rid of this.